omero_figure/scripts/omero/figure_scripts/Figure_To_Pdf.py

```diff
@@ -1294,6 +1294,7 @@ def draw_labels(self, panel, page):
         y = panel['y']
         width = panel['width']
         height = panel['height']
+        border = panel['border']
```

Comment:
better to use `border = panel.get('border')` which won't cause an error if border is missing